src/modules/pageTable.js

```diff
@@ -12,12 +14,17 @@ class PageTableRow extends React.PureComponent {
     }
 
     render() {
-        const { columns, isClickable, row } = this.props;
+        const {
+            columns,
+            idPrefix,
+            row,
+            rowIndex,
+            splitter,
+        } = this.props;
+        const sizes = this.props.sizes || [];
 
         return (
-            <Table.Row
-                onClick={isClickable ? this._onClick : null}
-            >
+            <Table.Row onClick={this._onClick}>
```

Comment:
Did we lose `isClickable` check here deliberately?  I guess you're thinking that it is handled in `_onClick()` so no need to conditionally bind a function or not here?

Reply:
I wanted to implement scroll right for non sticked columns on click - I've tested again, it works badly. So, I've restored  `isClickable` as it was.